spec.bs

```diff
@@ -2663,11 +2660,18 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|, a
   1. Let |igAd| be the [=interest group ad=] from |winner|'s [=generated bid/interest group=]'s
     [=interest group/ads=] whose [=interest group ad/render url=] is |winner|'s
     [=generated bid/ad descriptor=]'s [=ad descriptor/url=].
-  1. If |igAd|'s [=interest group ad/buyer and seller reporting ID=]
-     [=map/exists=] and the result of [=query reporting ID k-anonymity count=]
-     given |winner|'s [=generated bid/interest group=] and |igAd| is true, then [=map/set=]
-     |browserSignals|["{{ReportingBrowserSignals/buyerAndSellerReportingId}}"] to |igAd|'s
-     [=interest group ad/buyer and seller reporting ID=].
+  1. If the |winner|'s [=generated bid/selected buyer and seller reporting ID=] is not null:
+    1. [=Assert=] that the result of running [=query reporting ID k-anonymity count=] with |winner|'s
+      [=generated bid/interest group=], |igAd|, and |winner|'s
+      [=generated bid/selected buyer and seller reporting ID=] is true.

```

Comment:
It might be worth adding a note here that this is expected at this point because any bid that included a selectedBuyerAndSellerReportingID wouldn't be considered k-anonymous, and thus couldn't win the auction and wouldn't be reported, unless that bid was k-anonymous for reporting with that selectedBuyerAndSellerReportingID.

I don't see a lot of other notes in the spec, but occasionally they seem to be expressed on the latter half of the same item, so perhaps ...

```
    1. [=Assert=] that the result of running [=query reporting ID k-anonymity count=] with |winner|'s
      [=generated bid/interest group=], |igAd|, and |winner|'s
      [=generated bid/selected buyer and seller reporting ID=] is true.

      Note: The ad is expected to be k-anonymous for reporting because any bid that included a
      selectedBuyerAndSellerReportingID wouldn't be considered k-anonymous, and thus couldn't
      win the auction and wouldn't be reported, unless that bid was k-anonymous for reporting
      with that selectedBuyerAndSellerReportingID.
```